rs/consensus/src/idkg/test_utils.rs

```diff
@@ -87,8 +87,8 @@ pub(crate) fn empty_response() -> ic_types::batch::ConsensusResponse {
     )
 }
 
-fn fake_signature_request_args(key_id: MasterPublicKeyId) -> ThresholdArguments {
-    match key_id {
+fn fake_signature_request_args(key_id: IDkgMasterPublicKeyId) -> ThresholdArguments {
```

Comment:
Note that if we decide to extend the `SignWithThresholdContext` for VetKd, instead of introducing a new context, passing a `MasterPublicKeyId` here probably makes sense.